Off-Top-App/off-top-flutter
# OT-SettingsLogout | Added a logout button and slightly redesigned settings page

**What I Did:** 

- Added a new feature that allows a user to log out from their signed in account.

- Changed settings_page.dart to follow what Figma has designed for the settings page. 

- Fixed an error with Google Services for a mistake I made when telling David to cast a variable as something else. Settings and login/logout should work 100% now.

**How To Test:**

- Prerequisites: You will need to run `flutter pub get` to pull the new packages used for the sign out.

- You will also need Zookeeper, Kafka, and Spring-boot servers running to log into the app.

- Go to the settings page, make sure that `Preferences` will open with a tap.

- Tap `Preferences` again to exit out of it and then test the `Sign Out` button. This should bring you back to the login page and next time you sign in, it should prompt you for your Google Account again. 

- Test on both Android and iOS.
